src/Illuminate/Mail/Transport/MailgunTransport.php

```diff
@@ -93,23 +93,63 @@ public function send(Swift_Mime_SimpleMessage $message, &$failedRecipients = nul
      */
     protected function payload(Swift_Mime_SimpleMessage $message, $to)
     {
-        return [
-            'auth' => [
-                'api',
-                $this->key,
-            ],
-            'multipart' => [
-                [
-                    'name' => 'to',
-                    'contents' => $to,
+        if (!config('services.mailgun.batch_sending') || count($message->getTo()) === 1) {
```

Comment:
Can't call foundation config helper from the mail component.

Reply:
Fixed, please review.